src/Uno.UI/UI/Xaml/Media/RadialGradientBrush.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Windows.Foundation;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Markup;
+
+namespace Windows.UI.Xaml.Media
+{
+	public sealed partial class RadialGradientBrush : GradientBrush
```

Comment:
Also the default value for `MappingMode` is `RelativeToBoundingBox`.

https://github.com/microsoft/microsoft-ui-xaml/blob/ca95be6f5248eaa55ec523f2d6e7b50f6e0d53e7/dev/RadialGradientBrush/RadialGradientBrush.idl#L28

Reply:
It's already the case... Check in `GradientBrush.cs`.